### PR TITLE
Clarify how fence_kdump works

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

Hopefully this makes the fence_kdump example clearer. 

### PR creator: Are there any relevant issues/feature requests?

* bsc#1228931
* jsc#DOCTEAM-537

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP6
  - [x] 15 SP5
  - [x] 15 SP4
  - [x] 15 SP3
  - [x] 15 SP2
- SLE-HA 12
  - [x] 12 SP5

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by tahliar** [comment]
image
image

File: xml/ha_fencing.xml [hunk 1]
```diff
@@ -446,56 +446,62 @@ hostlist</screen>
...
-    timeout="60"</command>
+    timeout="60"</command><co xml:id="co-ha-fence-kdump-timeout"/>
 &prompt.crm.conf;<command>commit</command></screen>
     <calloutlist>
-     <callout arearefs="co-ha-fenc-kdump-nodename">
+     <callout arearefs="co-ha-fence-kdump-nodename">
       <para>
        Name of the node to be monitored. If you need to monitor more than one
        node, configure more &stonith; resources. To prevent a specific node
        from using a fencing device, add location constraints.
```
Comment (zzhou1) [inline xml/ha_fencing.xml:483]:
```suggestion
       Name of the node to listen a message from <literal>fence_kdump_send</literal>. 
       Configure more &stonith; resources for other nodes if need.
```

File: xml/ha_fencing.xml [hunk 2]
```diff
@@ -446,56 +446,62 @@ hostlist</screen>
...
       </para>
      </callout>
+     <callout arearefs="co-ha-fence-kdump-timeout">
+       <para>
+        Defines how long to wait for a message from <literal>fence_kdump_send</literal>.
+        If a message is received, then a Kdump is in progress and the fencing mechanism
+        considers the node to be fenced. If no message is received during this timeout,
+        then no Kdump is in progress and the &stonith; device fences the node.
```
Comment (zzhou1) [inline xml/ha_fencing.xml:491]:
```suggestion
        considers the node to be fenced. If no message is received, then fence_dump get 
        timeout and that indicates the fence operation failed. The next &stonith;  
        device in the fence_topology fences the node eventually.
```

File: xml/ha_fencing.xml [hunk 3]
```diff
@@ -446,56 +446,62 @@ hostlist</screen>
...
+     Use the <literal>stonith:fence_kdump</literal> resource agent
+     to monitor all nodes with the Kdump function enabled. A
+     configuration example is shown below. For more information,
+     see <command>crm ra info stonith:fence_kdump</command>.
     </para>
 <screen>&prompt.root;<command>crm configure</command>
 &prompt.crm.conf;<command>primitive st-kdump stonith:fence_kdump \
-    params nodename="&node1; "\ </command><co xml:id="co-ha-fenc-kdump-nodename"/>
+    params nodename="&node1; "\ </command><co xml:id="co-ha-fence-kdump-nodename"/>
     <command>pcmk_host_check="static-list" \
```
Comment (zzhou1) [inline xml/ha_fencing.xml:472]:
```suggestion
    pcmk_host_list="&node1;" \
    <command>pcmk_host_check="static-list" \
```

REF: bsc#1007422

pcmk_host_list is documented in 
https://clusterlabs.org/pacemaker/doc/2.1/Pacemaker_Explained/singlehtml/#special-instance-attributes-for-fencing-resources

File: xml/ha_fencing.xml [hunk 4]
```diff
@@ -446,56 +446,62 @@ hostlist</screen>
...
     <step>
     <para>
-     Use the <literal>stonith:fence_kdump</literal> resource agent (provided
-     by the package <package>fence-agents</package>)
-     to monitor all nodes with the Kdump function enabled. Find a
-     configuration example for the resource below:
+     Use the <literal>stonith:fence_kdump</literal> resource agent
+     to monitor all nodes with the Kdump function enabled. A
+     configuration example is shown below. For more information,
+     see <command>crm ra info stonith:fence_kdump</command>.
```
Comment (zzhou1) [inline xml/ha_fencing.xml:467]: This sentence was legacy long ago and doesn't fit nowadays for sure.

```suggestion
     Use the <literal>stonith:fence_kdump</literal> fence agent. 
     A configuration example is shown below. For more information,
     see <command>crm ra info stonith:fence_kdump</command>.
```

**Review by zzhou1 — CHANGES_REQUESTED** [review]

**Comment by tahliar** [comment]
Thanks @zzhou1! I applied your fixes and made a couple of little edits. Let me know if you're happy with it :)

**Review by zzhou1 — APPROVED** [review]
Thx, Tahlia!

**Review by dariavladykina — APPROVED** [review]
LGTM!

**Pull request merged**